gauge.go

```diff
@@ -32,6 +32,18 @@ func (m *CirconusMetrics) RemoveGauge(metric string) {
 	delete(m.gauges, metric)
 }
 
+// GetGaugeTest returns the current value for a gauge. (note: it is a function specifically for "testing", disable automatic submission during testing.)
+func (m *CirconusMetrics) GetGaugeTest(metric string) (string, error) {
```

Comment:
Should this return a `uint64` instead of a string? Else I need to `strconv` the result in some cases (or cast the test value as a string).